app/components/tweet/model.js

```diff
@@ -0,0 +1,69 @@
+'use strict';
+
+const _get = require('lodash/get'),
```

Comment:
This is really picky, but I want to refactor `lodash` out of this project. Could we replace `_get` a local function that would check the value we need?